translations/Polish/special-terms.md

```diff
@@ -0,0 +1,41 @@
+# Terms
+English word | Polish word 
+
+* Message - Wiadomość
+* Private Message (PM) - Prywatna wiadomość
+* Group PM - Czat grupowy (Konwersacja would be too formal)
+* Realm - Domena (Królestwo wouldn't fit the context)
+* Stream - Kanał (Strumień wouldn't fit the context)
+* Topic - Wątek 
+* Private/Invite-Only Stream - Prywatny kanał (Kanał na zaproszenie seems too long)
+* Public Stream - Publiczny kanał
+* Bot - Bot (Robot just doesn't sound right in my opinion)
+* Integration  - Integracja
+* Notification - Powiadomienie
+* Alert Word - Słowo alarm
+* View - Widok
+* Home - Strona głowna
+* Emoji - Emotka (singular) or Emotki (plural)
+
+# Phrases
+
+* Subscribe to a stream/Unsubscribe from a stream - Zasubskrybuj kanał
+* Narrow to - Zwęzić do
+* Filter - Filtr
+* Mute/Unmute - Wycisz / Anuluj wyciszenie
+* Deactivate/Reactivate - Dezaktywuj/Reaktywuj
+* Search - Wyszukaj
+* Pin - Odznaka / Kołek (Depending on the context)
+* Mention/@mention - Wspomnij/@oznacz
```

Comment:
Mention is a verb here, I think `wspomnieć` or `oznaczyć` would work. `@mention` is more of a noun.